merkle_tree/src/prelude.rs

```diff
@@ -52,6 +54,36 @@ pub type RescueLightWeightMerkleTree<F> = LightWeightMerkleTree<F, RescueHash<F>
 /// Example instantiation of a SparseMerkleTree indexed by I
 pub type RescueSparseMerkleTree<I, F> = UniversalMerkleTree<F, RescueHash<F>, I, 3, F>;
 
+// TODO: (alex) move this compression to CRHF and wrap with better API?
```

Comment:
Agree that we should have a CRHF impl (and maybe sponge as well?

Reply:
Yeah, let's add in a separate PR. This is already getting big